app/views/shelters/index.html.erb

```diff
@@ -13,13 +13,14 @@
       <p><a class="edit" href="/shelters/<%= shelter.id %>/edit">Update Shelter</a></p>
       <p><a class="edit" href="/shelters/<%= shelter.id %>/pets/new">Create Pet</a></p>
       <% if shelter.pets.count == 0 %>
-      <section class="delete-form">
-        <%= button_to "Delete #{shelter.name}", "/shelters/#{shelter.id}", method: :delete %>
-      </section>
+        <section class="delete-form">
+          <%= button_to "Delete #{shelter.name}", "/shelters/#{shelter.id}", method: :delete %>
+        </section>
```

Comment:
if you want to format this puppy, you can add `class: "grey hover"` after `:delete`